projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.html

```diff
@@ -3,89 +3,179 @@ <h2 class="heading-headline py-3 mt-3 mb-5 text-center">
 </h2>
 <ng-container *ngIf="quotes$ | async as quotes">
   <ng-container *ngIf="quotesLoaded$ | async; else loading">
-    <div class="row" *ngIf="quotes.length > 0; else noQuotes">
-      <div *ngFor="let quote of quotes" class="col-12 col-md-6 d-flex">
-        <div class="info-card">
-          <h6 class="title background-color-4">
-            {{ quote.defaultCategory?.name }}
-          </h6>
-          <div class="info-card-wrapper">
-            <div class="info-card-content">
-              <div class="info-card-details">
-                <div class="info-card-data">
-                  <span class="label">{{ quote.planName }}</span
-                  >:
-                  <span class="value">{{ quote.quoteId }}</span>
-                </div>
-                <ng-container *ngIf="quote.quoteWorkflowStatus">
+    <div *ngIf="quotes.length > 0; else noQuotes">
+      <div class="row justify-content-between align-items-center mb-5">
+        <div class="col-md-5 d-flex">
+          <label class="col-8 pr-3 px-0">
+            <span class="label-content">{{
+              'quote.productCategory' | cxTranslate
+            }}</span>
+            <ng-select
+              placeholder="{{ 'quote.selectCategory' | cxTranslate }}"
+              [searchable]="false"
+              [clearable]="false"
+              [items]="options"
+              bindLabel="name"
+              bindValue="code"
+              (change)="selectCategory($event)"
+            >
+            </ng-select>
+          </label>
+          <button
```

Comment:
button should be wrapped in `col-4` and have only `btn-link link` classes itself

Reply:
If I remove `align-self-end pb-4` classes from button I get:

![Screenshot 2021-09-14 at 15 52 40](https://user-images.githubusercontent.com/75082466/133270370-f994d97f-9802-4870-9465-6489aac54224.png)

Reply:
Have you wrapped button in additional div with the class `col-4` as I suggested?

Reply:
No, I didn't read well, I added that class on the button. With wrapper works fine.